.github/workflows/release.yml

```
@@ -7,18 +7,38 @@ on:
 
 jobs:
   build:
-    name: Container build
+    name: Build and push image
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Publish
-        run: ./ci/release.sh
-        env:
-          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
-          RELEASE_TAG: ${{ github.ref }}
-      - name: Run Snyk monitor docker image
-        run: ./ci/snyk-scan.sh
-        env:
-          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
-          RELEASE_TAG: ${{ github.ref }}
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.16'
+
+      # The DOCKER_IMAGE_TAG var is generated here from the release tag, after some validation
+      - name: Generate docker image version from git tag
+        run: |
+          echo "${{ github.event.release.tag_name }}" | grep -E '^v[0-9.]*[0-9]$'
```

in this repo tags follow the 0.0.0 syntax
```suggestion
          echo "${{ github.event.release.tag_name }}" | grep -E '^[0-9.]*[0-9]$'
```